### ISSUE TITLE
How to contact the train and val dataset?

### ISSUE THREAD

**Issue by chengjianhong** [MedMNIST/MedMNIST#16]
If I need to contact the two data sets (training set and validation set) as whole training, how to do it? When I use ConcatDataset provided by Pytorch, the concatenated data can't return "imgs" and "labels". For example:

`train_data = data.ConcatDataset([train_dataset,val_dataset])`

the train_data can't directly get the "imgs" and "labels", such as "train_data .imgs" and "train_data .labels".

**Comment by chengjianhong** [comment]
I have solved it by modifying the code. Thanks.

**Comment by duducheng** [comment]
@chengjianhong actually you can check the PyTorch API to get the concat dataset.
